otawslibs/generate_aws_session.py

```diff
@@ -1,9 +1,20 @@
 import boto3
 
-def _create_session(aws_profile=None):
+def _create_session(aws_profile=None, role_arn=None):
```

Comment:
Discuss with Rajat/Abhishek rather having 2 variables can we have something like tuple/map/dictionary to have one single argument